integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorWlsUpgrade.java

```diff
@@ -532,5 +542,55 @@ private void createDomainResource(
          String.format("Create domain custom resource failed with ApiException "
              + "for %s in namespace %s", domainUid, domainNamespace));
     setPodAntiAffinity(domain);
+    removePortForwardingAttribute(domainNamespace,domainUid);
```

Comment:
can you make this call outside after installOperator method is called, move it from here as its not part of operator installation.

Reply:
removePortForwardingAttribute() method call is part of createDomainResource() --> createWlsDomainAndVerify() not part of Operator Installation